packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -237,13 +237,17 @@ const makeRenderer = (
 		const userAgent = headers['user-agent'];
 		const userAgentDevice = headers['x-ua-device'] || ''; // set by fastly
 
+		// service worker can be disabled per-request with __disable_sw querystring param
+		const enableServiceWorkerReq =
+			'__disable_sw' in request.query || enableServiceWorker;
```

Comment:
Looks to me like having `__disable_sw` in a query string is in fact enabling service worker, setting `enableServiceWorkerReq` to `true`. Should probably be negated and `&&`-ed with config parameter.

Reply:
:facepalm: